roadmap.md

```diff
@@ -0,0 +1,44 @@
+# WebDriver BiDi roadmap
+
+## Real-world end-to-end user scenarios
+
+This document presents an overview of real-world end-to-end user scenarios we aim to enable via the WebDriver BiDi protocol. Each scenario requires one or more WebDriver BiDi commands and events to be specified, tested, and implemented across browser engines.
+
+### A. Extracting headlines
```

Comment:
I would suggest to use numbers given that we might have more than 24 scenarios in the end. Do we want to make use of `Milestones`? If not maybe we want to finally transfer all these scenarios to real milestones eventually. Would be good to also get @jgraham's opinion given that he wasn't present at the WG meeting.

Note that the current list of milestones doesn't match with those that we finally decided on to work. You may want to have a look at the meeting notes again.

Eg. the logging feature was the API that we supported and I feel as such it should come first. Also it's the first thing the Selenium project implemented for BiDi support.

Reply:
> I would suggest to use numbers given that we might have more than ~24~ _26_ scenarios in the end.

Happy to use whatever numbering scheme. I think running out of digits/letters might not really become a problem in practice — especially since one the suggestions during the meeting was to keep the list of e2e scenarios relatively small: https://www.w3.org/2022/08/24-webdriver-minutes.html#t02:~:text=simonstewart%3A%20my%20preference%20would%20be%20define%20top%203%2C%20and%20when%20one%20is%20done%2C%20add%20another

> Note that the current list of milestones doesn't match with those that we finally decided on to work. You may want to have a look at the meeting notes again.

To clarify, the current patch is a simplified version with some example e2e scenarios based on our own roadmap, in a format I think might be useful here. I do not intend for this to be the final version! With this PR I merely wanted to kick off the discussion.

If this format looks good to everyone, we can start thinking of e2e scenarios that capture the remaining pieces mentioned in the meeting notes: https://www.w3.org/2022/08/24-webdriver-minutes.html#t02:~:text=Updated%20list%3A,*%20printing I’m hoping for the WG’s input to help come up with clear & concise e2e use cases. E.g. @whimboo, what would be an easy-to-understand e2e example involving logging?

Reply:
As discussed I was working on enhancements for this PR locally. I'll have an update soon that I can share. Closing this issue for now.